### DISCUSSION TITLE
Eligibility Inquiry for GitHub Campus Expert Application

### DISCUSSION THREAD

**Discussion by YounesBensafia** [community/community#148035] [GitHub Education] [answered]
### Hi everyone,

Hello,
I am currently in the first year of my Master's program (Master 1) and plan to **graduate in 2026**. I intend to apply to become a GitHub Campus Expert in **July 2025**. Could you please confirm if I am eligible to apply, given that I will still have more than one year of study remaining at that time?

**Comment by davevad93** [comment, marked as answer]
Hi @YounesBensafia ,

Given the [eligibility criteria](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert#eligibility-criteria) stated in the docs, you need to have more than one year as a student before you graduate, so technically speaking you shouldn't be eligible if you plan to do so in the second half of **2025**, also note that applications for the program open in **February** and **August**, with a full month available to apply.

**Reply by YounesBensafia** [reply]
Actually, I checked [here](https://education.github.com/campus_experts) , and it says applications will open in July 2025 :')

**Reply by davevad93** [reply]
You're right @YounesBensafia it looks like in 2025 the applications will open in July, I guess the [docs](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/use-github-at-your-educational-institution/applying-to-be-a-github-campus-expert#eligibility-criteria) need an update.
